content/blog/2021-01-15-balancing-reuse-and-duplication-with-react/index.mdx

```diff
@@ -0,0 +1,182 @@
+---
+slug: balancing-reuse-and-duplication-with-react
+date: 2021-01-15
+title: 'Balancing Reuse and Duplication with React'
+published: true
+banner: ./undraw_logic.png
+---
+
+Working with
+[React Components](https://reactjs.org/docs/components-and-props.html) in a
+project I frequently come across things that are kind of like a component that
```

Comment:
Can we reword this? 

It reads (to me) as "I frequently come across things that are kind of like a component..."

When I think what you are trying to say is "sometimes components already exist, but have a slight variation that changes the behavior just enough"